modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1385,7 +1390,7 @@ This command uploads an asset description page from the path passed in `<mdPath>
 Naming the page "home" makes the uploaded page the main description page for the Exchange asset.
 
 [NOTE]
-This command only supports published pages.
+This command will publish any active draft as part of the operation.
 
 Argument `assetIdentifier` should be formatted as follows: `([group_id]/)<asset_id>/<version>`. +
```

Comment:
Why is `group_id` identified with `(` and `[`? 
Can this be formatted `<group_id>/<asset_id>/<version>` as shown in the [exchange asset upload](https://docs.mulesoft.com/runtime-manager/anypoint-platform-cli-commands#exchange-asset-upload) command?

I see other published instances where we document:
> ([groupID]/)assetID/version

Is there a difference between them?

Reply:
It's an optional argument. 

```
Argument "assetIdentifier" should be formatted as follows: ([group_id]/)<asset_id>/<version>. If "group_id" is not specified, it will default to the currently selected Organization ID.
```

Reply:
Thank you. 
In that case, I think it would make more sense to go with 
> `[group_id]/<asset_id>/<version>`

At least from the docs we never followed a convention about how to denote optional attributes. But overall it tends to be accepted that `[]` denotes "optional", while `<>` denotes "required".